pkg/registry/server/server.go

```diff
@@ -461,8 +462,8 @@ func MetadataEntriesFromQuery(w http.ResponseWriter, r *http.Request) ([]api.Met
 		if variantsExist {
 			element.Variant = variants[i]
 		}
-		metadata = append(metadata, element)
+		metadata.Insert(element)
 	}
 
-	return metadata, nil
+	return metadata.UnsortedList(), nil
```

Comment:
Ok, it's not sorted anymore but it doesn't matter.

Reply:
yes, the order doesn't matter. We don't have any unit tests here anyways. If we did, we would probably need to guarantee the order.